spec/higher_level_api/integration/connection_spec.rb

```diff
@@ -46,6 +46,30 @@
         expect { described_class.new("amqp://dev.rabbitmq.com/a/path/with/slashes") }.to raise_error(ArgumentError)
       end
     end
+
+    context "when URI contains query parameters" do
+      subject(:session) do
+        described_class.new("amqps://bunny_gem:bunny_password@/bunny_testbed?heartbeat=10&connection_timeout=100&channel_max=1000&verify=false&cacertfile=spec/tls/ca_certificate.pem&certfile=spec/tls/client_certificate.pem&keyfile=spec/tls/client_key.pem")
+      end
+
+      it "parses extra connection parameters" do
+        # session.start # raises "OpenSSL::SSL::SSLError: hostname "127.0.0.1" does not match the server certificate"
```

Comment:
https://groups.google.com/d/msg/rabbitmq-users/V4fCdASYT1I/75TG_VhVCwAJ

Reply:
I responded there.

Reply:
Inspecting the transport suggests that it has `:ssl=>0`.